src/it/scala/com/exasol/cloudetl/BaseS3IntegrationTest.scala

```diff
@@ -0,0 +1,84 @@
+package com.exasol.cloudetl
+
+import java.io.File
+
+import com.exasol.dbbuilder.dialects.Table
+
+import com.amazonaws.services.s3.model._
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import org.apache.hadoop.fs.{Path => HPath}
+import org.testcontainers.utility.DockerImageName
+import org.testcontainers.containers.localstack.LocalStackContainer
+import org.testcontainers.containers.localstack.LocalStackContainer.Service.S3
+
+trait BaseS3IntegrationTest extends BaseIntegrationTest {
+
+  val LOCALSTACK_DOCKER_IMAGE = DockerImageName.parse("localstack/localstack:0.12.5")
+  val s3Container = new LocalStackContainer(LOCALSTACK_DOCKER_IMAGE)
+    .withServices(S3)
+    .withReuse(true)
+  var s3: AmazonS3 = _
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    s3Container.start()
+    prepareS3Client()
+  }
+
+  override def afterAll(): Unit = {
+    s3Container.stop()
+    super.afterAll()
+  }
+
+  def prepareS3Client(): Unit =
```

Comment:
This is the sed v1 client. Maybe you should consider updating to AWS sdk v2 somewhen

Reply:
I think it comes with dependency, either with `localstack` or `hadoop-aws`. Both of them are latest versions. I am not going to add another dependency for it at the moment.